src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -4,16 +4,16 @@
 public class Task2 {
     public static String firstTwo(String s) {
 
-        if (s.length()<2)
-            return s;
+        if (s == null)
+            return null;
 
-        if (s.equals(""))
-            return "";
+        if (s.length() < 2)
+            return s;
 
         else return (new StringBuilder().append(s.charAt(0)).append(s.charAt(1))).toString();
```

Comment:
there is no need for StringBuilder usage here. just use `String.substring()` method